testsys/src/run_aws_k8s.rs

```diff
@@ -129,6 +129,10 @@ pub(crate) struct RunAwsK8s {
     #[structopt(long)]
     keep_instance_provider_running: bool,
 
+    /// Allow the sonobuoy test agent to rerun failed test.
+    #[structopt(long)]
+    allow_retries: Option<u32>,
```

Comment:
Not sure my suggestion is the best but `--allow-retries` sounds like a boolean.

```suggestion
    retry_failed_attempts: Option<u32>,
```